x/bank/types/querier.go

```diff
@@ -57,3 +57,35 @@ type QuerySupplyOfParams struct {
 func NewQuerySupplyOfParams(denom string) QuerySupplyOfParams {
 	return QuerySupplyOfParams{denom}
 }
+
+// TODO HV2: from bank/querier.go in heimdall repo; might not be needed, double check
```

Comment:
I would be careful and make sure to make core modules like bank as consistent as possible with the upstream sdk. This is what it should look like outside of any custom logic: https://github.com/cosmos/cosmos-sdk/blob/release/v0.50.x/x/bank/types/querier.go

Reply:
Additionally the constructor function for the bank grpc querier has been moved to the keeper https://github.com/cosmos/cosmos-sdk/blob/release/v0.50.x/x/bank/keeper/grpc_query.go#L25

Reply:
Yeah agreed. The comment's point was to get clarified whether we need to do any modifications on querier but it seems grpc does the job. I'll remove it.